File: 프로그래머스/unrated/120840. 구슬을 나누는 경우의 수/구슬을 나누는 경우의 수.py
def solution(balls, share):
    n = 1;
    for i in range(balls, 0, -1):
        if i == share: break;
        n *= i;
    m = 1;
    ran = balls - share;
    for j in range(ran, 0, -1):
        m *= j;
    return round(n / m)
        
   

File: 프로그래머스/unrated/120840. 구슬을 나누는 경우의 수/구슬을 나누는 경우의 수.js
function solution(balls, share) {
    let n = 1;
    for(let i = balls; i >= 1; i--){
        if(i === share) break;
        n *= i;
    }
    
    let m = 1;
    for(let i = balls - share; i >= 1; i--){
        m *= i;
    }
    
    return Math.floor(n / m)
}